ui/app/templates/components/clients/history.hbs

```diff
@@ -190,7 +190,7 @@
       <div class="modal-radio-button">
         <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
           <D.Trigger
-            data-test-popup-menu-trigger="true"
```

Comment:
this is also missing from the original pr?

![image](https://user-images.githubusercontent.com/6618863/171969710-1beb33c6-a2f6-41a3-95f7-08b1b5be21d1.png)

Reply:
There are some file changes here that don't show on the original PR and some that are only on the original and not this pr (specifically for this file)?

Reply:
Same comment as above, from original PR:
<img width="469" alt="Screen Shot 2022-06-06 at 10 38 38 AM" src="https://user-images.githubusercontent.com/68122737/172215149-190c9d70-ecc7-41b2-ad16-1bf391460691.png">